rocketpy/AeroSurfaces.py

```diff
@@ -1918,3 +1916,55 @@ def allInfo(self):
         self.aerodynamicInfo()
 
         return None
+
+
+class RailButtons:
```

Comment:
It seems weird to me to place the RailButton class inside the AeroSurfaces modules. How are these related?